modules/oadp-operator-issues.adoc

```diff
@@ -0,0 +1,93 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/troubleshooting.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="oadp-operator-issues_{context}"]
+= OADP Operator issues
+
+You might encounter issues caused by problems with the {oadp-first} Operator.
```

Comment:
```suggestion
The {oadp-first} Operator may encounter problems it is not able to resolve.
```

Reply:
Fixed, though I stuck with "might" as per the IBM guide.